docs/reference/esql/functions/to_cartesianpoint.asciidoc

```diff
@@ -0,0 +1,32 @@
+[discrete]
+[[esql-to_cartesianpoint]]
+=== `TO_CARTESIANPOINT`
+Converts an input value to a `point` value.
+
+The input can be a single- or multi-valued field or an expression.
+The input type must be a string or long integer.
+
+A string will only be successfully converted if it's respecting the
+https://en.wikipedia.org/wiki/Well-known_text_representation_of_geometry[WKT Point] format:
+
+[source.merge.styled,esql]
+----
+include::{esql-specs}/spatial.csv-spec[tag=to_cartesianpoint-str]
+----
+[%header.monospaced.styled,format=dsv,separator=|]
+|===
+include::{esql-specs}/spatial.csv-spec[tag=to_cartesianpoint-str-result]
+|===
+
+If the input parameter is of numeric type `long`, its value will be interpreted as
+the Lucene encoded value, with `x` encoded in the high four bytes,
```

Comment:
This capability is very useful internally and for testing, but it is not clear to me that it does need to be exposed. It will become even more useful when we have both source and doc-values versions of the block data, each with specific purposes. For now, we could either remove it from the docs, or leave it there for now while the system is still in tech-preview, or keep it if in the end we want to expose it permanently.

Reply:
I don't think we should accept longs anyway